bedrock/firefox/templates/firefox/new/wait-face/base.html

```diff
@@ -12,6 +12,12 @@
   {% endif %}
 {% endblock %}
 
+{% block experiments %}
+  {% if switch('experiment-firefox-new-waitface'), ['en-US'] %}
```

Comment:
Wow, this is a sneaky bug. The locale here is not applied, and no error is thrown, because the close paren comes too early.

Reply:
Sneaky indeed, nice spot 👍